lib/bugsnag/railtie.rb

```diff
@@ -45,8 +45,10 @@ class Railtie < Rails::Railtie
         ActionController::API.send(:include, Bugsnag::Rails::ControllerMethods)
       end
       if defined?(ActiveRecord::Base)
-        require "bugsnag/rails/active_record_rescue"
-        ActiveRecord::Base.send(:include, Bugsnag::Rails::ActiveRecordRescue)
+        ActiveSupport.on_load(:active_record) do
```

Comment:
@celsodantas pointed out this is probably unacceptable because it creates a dependency on active support :(.